docs/traffic-control.md

```diff
@@ -0,0 +1,277 @@
+
+### TargetPort
+
+The `targetPort` field specifies the port to which the traffic should be
+redirected or mirrored. There are five kinds of ports that can be used to
+receive mirrored traffic:
+
+**ovsInternal**: This specifies an OVS internal port on all Nodes. A Pod's
+traffic will be redirected or mirrored to the OVS internal port on the same Node
+that hosts the Pod. The port doesn't need to exist in advance, Antrea will
+create the port if it doesn't exist. To use an OVS internal port, the `name` of
+the port must be provided:
+
+```yaml
+ovsInternal:
+  name: tap0
+```
+
+**device**: This specifies a network device on all Nodes. A Pod's traffic will
+be redirected or mirrored to the network device on the same Node that hosts the
+Pod. The network device must exist on all Nodes and Antrea will attach it to the
+OVS bridge if not already attached. To use a network device, the `name` of the
+device must be provided:
+
+```yaml
+device:
+  name: eno2
+```
+
+**geneve**: This specifies a remote destination for a GENEVE tunnel. All
+selected Pods' traffic will be redirected or mirrored to the destination via
+a GENEVE tunnel. The `remoteIP` field must be provided to specify the IP address
+of the destination. Optionally, the `destinationPort` field could be used to
+specify the UDP destination port of the tunnel, or 6081 will be used by default.
+If Virtual Network Identifier (VNI) is desired, the `vni` field can be specified
+to an integer in the range 0-16,777,215:
+
+```yaml
+geneve:
+  remoteIP: 10.0.10.2
+  destinationPort: 6081
+  vni: 1
+```
+
+**vxlan**: This specifies a remote destination for a VXLAN tunnel. All
+selected Pods' traffic will be redirected or mirrored to the destination via
+a VXLAN tunnel. The `remoteIP` field must be provided to specify the IP address
+of the destination. Optionally, the `destinationPort` field could be used to
+specify the UDP destination port of the tunnel, or 4789 will be used by default.
+If Virtual Network Identifier (VNI) is desired, the `vni` field can be specified
+to an integer in the range 0-16,777,215:
+
+```yaml
+vxlan:
+  remoteIP: 10.0.10.2
+  destinationPort: 4789
+  vni: 1
+```
+
+**gre**: This specifies a remote destination for a GRE tunnel. All selected
+Pods' traffic will be redirected or mirrored to the destination via a GRE
+tunnel. The `remoteIP` field must be provided to specify the IP address of the
+destination. If GRE key is desired, the `key` field can be specified to an
+integer in the range 0-4,294,967,295:
+
+```yaml
+gre:
+  remoteIP: 10.0.10.2
+  key: 1
+```
+
+**erspan**: This specifies a remote destination for an ERSPAN tunnel. All
+selected Pods' traffic will be mirrored to the destination via an ERSPAN tunnel.
+The `remoteIP` field must be provided to specify the IP address of the
+destination. If ERSPAN session ID is desired, the `sessionID` field can be
+specified to an integer in the range 0-1,023. The `version` field must be
```

Comment:
We do not have a default value for `version`?

Reply:
@jianjuns We don't have a default value for `version` since it is not a pointer type in API, and it is int type.

Reply:
Ok